config/environments/development.rb

```diff
@@ -14,6 +14,8 @@
   config.action_controller.perform_caching = false
 
   # Don't care if the mailer can't send.
+  config.action_mailer.default_url_options = { :host => 'localhost:3000' }
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.